velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -245,6 +245,37 @@ TEST_F(StringTest, ascii) {
   EXPECT_EQ(ascii(std::nullopt), std::nullopt);
 }
 
+TEST_F(StringTest, bitLength) {
+  auto bitLength = [&](std::optional<std::string> arg) {
+    return evaluateOnce<int32_t>("bit_length(c0)", arg);
+  };
+
+  EXPECT_EQ(bitLength(""), 0);
+  EXPECT_EQ(bitLength(std::string("\0", 1)), 8);
+  EXPECT_EQ(bitLength("1"), 8);
+  EXPECT_EQ(bitLength("123"), 24);
+  EXPECT_EQ(bitLength("😋"), 32);
+  // Consists of five codepoints.
+  EXPECT_EQ(bitLength(kWomanFacepalmingLightSkinTone), 136);
+  EXPECT_EQ(bitLength("\U0001F408"), 32);
+}
+
+TEST_F(StringTest, bitLengthVarbinary) {
+  auto bitLengthVarbinary = [&](std::optional<std::string> arg) {
```

Comment:
nit: now you can drop 'Varbinary' suffix since the lambda appears within bitLengthVarbinary test

Reply:
Updated.